V2/time_tracker/customers/_infrastructure/_data_persistence/_customer_dao.py

```diff
@@ -1,6 +1,8 @@
 import dataclasses
+import typing
 
 import sqlalchemy as sq
+import sqlalchemy.sql as sql
```

Comment:
as you import all sqlalchemy it should not import sql again you should only have` sq.sql.<nameImport>.`